configure.ac

```diff
@@ -492,6 +492,21 @@ if test -n "$LIBSELINUX" ; then
     LIBS=$BACKUP_LIBS
 fi
 
+AC_ARG_ENABLE([sandbox],
+    AS_HELP_STRING([--enable-sandbox],[Enable landlock sandbox]),
```

Comment:
I think it should be `--disable-sandbox` instead (i.e. secure by default), otherwise nobody will use sandboxing.

Reply:
BTW, **L**andlock is a proper noun. ;)

Reply:
No! It should not be enabled by default. At least not currently.

Reply:
Anyway,  `sandbox` is too generic, please use `landlock` instead.

Reply:
At least it needs to be part of the CI, with a kernel supporting Landlock (Linux >= 5.13, and potentially >= 5.19 to check `LANDLOCK_ACCESS_FS_REFER`).

Reply:
Done, the flat has been renamed and is disabled by default.